config/locales/social.yml

```diff
@@ -119,6 +119,10 @@ ru: # Russian
     instagram: https://www.instagram.com/wemeditate.ru
     vk: 'https://vk.com/wemeditate'
 
+sv: # Swedish
+  social_media:
+    instagram: ""
```

Comment:
You definitely should include the `sharing` block though right? Not sure, but I expect excluding it would cause an error on any article page or any other page with sharing links.

Reply:
Added this just to get the page to render :)

I think your latest refactoring fixed the underlying issue. Regarding social channels we won't have the manpower to manage special swedish ones just for wemeditate. Potentially could link them to Sahaja Yoga ones but not sure if appropriate.

Reply:
Most languages if they don't have their own just use the English ones, that's fine. Like Italian.